wpiutil/src/main/native/thirdparty/llvm/include/wpi/fs.h

```diff
@@ -40,7 +40,7 @@ using fstream = std::fstream;
 #ifndef GHC_USE_STD_FS
```

Comment:
fs.h/cpp doesn't really have anything in common with a llvm upstream file.  It's a custom file for WPILib with pieces from both LLVM and GHC filesystem.

Reply:
It mocks llvms `FileSystem.h`, therefore it feels like it should belong there

Reply:
My view is that if the diff is larger than the file itself, it's not "tracking" upstream.